src/main/pages/setup-openshift/openshift-admin-guide.adoc

```diff
@@ -407,23 +397,22 @@ grantMethod: prompt
 ')
 ----
 
-*Note*: Adding a OAuth client requires cluster-wide admin rights.
+See https://www.keycloak.org/docs/3.3/server_admin/topics/identity-broker/social/openshift.html[Keycloak documentation] for more information on the Keycloak OpenShift identity provider.
 
 [id="che-configuration"]
 ==== Che configuration
 
-On the Che deployment configuration:
+To configure Che deployment:
 
-* the `CHE_INFRA_OPENSHIFT_PROJECT` environment variable should be set to `NULL` to ensure a new distinct OpenShift namespace is created for every started workspace.
-* the `CHE_INFRA_OPENSHIFT_OAUTH__IDENTITY__PROVIDER` environment variable should be set to the alias of the OpenShift identity provider specified in step 1 of its link:#openshift-identity-provider-registration[registration in Keycloak]. The default value is `openshift-v3`.
+* Set the `CHE_INFRA_OPENSHIFT_PROJECT` environment variable to `NULL` to ensure a new distinct OpenShift namespace is created for every started workspace.
+* Set the `CHE_INFRA_OPENSHIFT_OAUTH__IDENTITY__PROVIDER` environment variable to the alias of the OpenShift identity provider specified in step 1 of its link:#openshift-identity-provider-registration[registration in Keycloak]. The default value is `openshift-v3`.
 
 [id="providing-the-openshift-certificate-to-keycloak"]
 ==== Providing the OpenShift certificate to Keycloak
 
 If the certificate used by the OpenShift console is self-signed or is not trusted, then by default the Keycloak will not be able to contact the OpenShift console to retrieve linked tokens.
 
-In this case the OpenShift console certificate should be passed to the Keycloak deployment as an additional environment property. This will enable the Keycloak server to add it to its list of trusted certificates, and will fix the problem.
-
-The environment variable is named `OPENSHIFT_IDENTITY_PROVIDER_CERTIFICATE`.
+Keycloak can not contact the OpenShift console to retrieve linked tokens when the certificate used by the OpenShift console is self-signed or is not trusted.
 
-Since adding a multi-line certificate content in a deployment configuration environment variable is not that easy, the best way is to use a secret that contains the certificate, and refer to it in the environment variable.
+When the certificate is self-signed or is not trusted, use the enivronment variable `OPENSHIFT_IDENTITY_PROVIDER_CERTIFICATE` to pass the OpenShift console certificate to the Keycloak deployment. This will enable the Keycloak server to add the certificate to the list of trusted certificates.
```

Comment:
* typo: enivronment
* better word order: "use the `OPENSHIFT_IDENTITY_PROVIDER_CERTIFICATE` environment variable"